tests/test_vpc.py

```diff
@@ -2689,3 +2689,34 @@ def test_vpc_delete_flow_logs(self):
             "FlowLogs"
         ]
         self.assertFalse(logs)
+
+    def test_vpc_set_flow_logs_maxaggrinterval(self):
+        session_factory = self.replay_flight_data("test_vpc_set_flow_logs_maxaggrinterval")
+        p = self.load_policy(
+            {
+                "name": "c7n-vpc-flow-logs-maxinterval",
+                "resource": "vpc",
+                "filters": [
+                    {"tag:Name": "FlowLogTest"}
```

Comment:
should have a flow log filter here to verify pre-conditions.

Reply:
Addressed, thanks!